index.html

```diff
@@ -1879,6 +1879,88 @@ <h3>Versioning Cryptography Suites</h3>
         </p>
       </section>
 
+      <section>
+        <h3>Prioritize Application Developers</h3>
```

Comment:
> [@msporny] The titles need to be 3 words long or they wrap on most devices, we want something pithy. The title you provided is accurate, but might not result in something "sticky".
>
> Suggesting "Prioritize Application Developers" as a "Security Consideration".

Hrm. That line wrap length is VERY limiting, and quite troubling! I'm revising my suggestion anyway. "Prioritize" still feels wrong. My revised suggestion of "Protecting Application Developers" doesn't feel quite right, either, as I think it's more like "Protecting Application Developers [and Users from Developer Footguns]"... Maybe saying this will spark inspiration in someone else.

```suggestion
        <h3>Protecting Application Developers</h3>
```